version/version.go

```diff
@@ -4,7 +4,7 @@ import "github.com/hashicorp/packer-plugin-sdk/version"
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "1.0.1"
+	Version = "2.0.0"
```

Comment:
Normally we wait until the commit that cuts the release to bump the version number, I am going to bump this right after merging though so I think this is fine, definitely agree we should bump the plugin to v2 though!